zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -146,15 +145,6 @@ object ServerSpec extends HttpRunnableSpec {
         val res  = Http.fromFile(file).requestBodyAsString()
         assertM(res)(equalTo("abc\nfoo"))
       } +
-      testM("content-type header on file response") {
-        val file = new File(getClass.getResource("/TestFile.txt").getPath)
-        val res  =
-          Http
-            .fromFile(file)
-            .requestHeaderValueByName()(HttpHeaderNames.CONTENT_TYPE)
-            .map(_.getOrElse("Content type header not found."))
-        assertM(res)(equalTo("text/plain"))
-      } +
```

Comment:
I think we should not break this functionality. ContentType is required even in `fromFile`.